var/spack/repos/builtin/packages/coreneuron/package.py

```diff
@@ -72,12 +73,13 @@ class Coreneuron(CMakePackage):
     depends_on('neurodamus-base@master', when='@master')
     depends_on('neurodamus-base@mousify', when='@mousify')
     depends_on('neurodamus-base@plasticity', when='@plasticity')
-    depends_on('neurodamus-base@hippocampus', when='@hippocampus')
+    depends_on('neurodamus-base@hippocampus', when='@hippocampus') 
 
-    # sympy and ispc options are only usable with nmodl
+    # sympy, codegen and ispc options are only usable with nmodl
     conflicts('+sympyopt', when='~sympy')
     conflicts('+sympy', when='~nmodl')
     conflicts('+sympy', when='coreneuron@0.17')  # issue with include directories
+    conflicts('~codegenopt', when='~nmodl')
```

Comment:
why this is a conflict ? I think we can simply remove this.

Reply:
it does not make sense to call +/~codegenopt without nmodl. It is an nmodl option...

Reply:
yeah thats true..but also its "legal" to have `~codegenopt` when `~nmodl` and hence was saying we can remove it. But anyway this is very minor thing, we can leave as it is.

Reply:
in this case it is  "legal" to have `+sympy` when `~nmodl` but we raise a conflict. In addition, if you wrote `~codegenopt` with `~nmodl` you wanted to do something else. Better to fail as soon as possible